scripts/mac_intel_install.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/zsh
+set -e
+# The purpose of the prefix is to give an indication of the point
+# of execution of this script, so that if something breaks it's easier
+# to see where that broke.
+prefix="[LM Install Script] "
+
+# Credit, Taken from: https://stackoverflow.com/a/34389425
+# Installs homebrew if it does not exist, or updates it if it does.
+which -s brew
+if [[ $? != 0 ]] ; then
+    echo "${prefix}Installing homebrew"
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+else
+    echo "${prefix}Updating homebrew"
+    brew update
+fi
+
+echo "${prefix}Installing node"
+brew install node
+
+echo "${prefix}Installing pyenv"
+brew install pyenv
+
+echo "${prefix}Installing git"
+brew install git
+
+echo "${prefix}Installing direnv"
+brew install direnv
+
+echo "${prefix}Installing postgres"
+brew install postgresql
+
+echo "${prefix}Installing libpq"
+brew install libpq
+
+echo "${prefix}Installing watchman"
+brew install watchman
+
+echo "${prefix}Installing gsl"
+brew install gsl
```

Comment:
Why do we need this?

Reply:
I don't think we do. It should probably come out of the README as well

Reply:
I needed it when I originally installed, if it's no longer needed I'll remove it